index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link
+    href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Gentium+Plus:wght@700&family=Poppins&family=Roboto:wght@700&display=swap"
+    rel="stylesheet">
+  <script src="https://kit.fontawesome.com/213f6776ae.js" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="./styles.css">
+</head>
+
+<body>
+  <header>
+    <div class="logo">Bright Ezeamaka</div>
```

Comment:
- [ ] [Optional] Kindly note that the logo (if the design has one) is on the left and has a link (`a` tag).
  - This is for three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.
 
E.g: 
```HTML  
<div class="logo">
   <a href="">Bright Ezeamaka</a>
</div>
```

>N.B: I actually marked this as optional because it wasn't requested by the previous reviewer but its part of the requirement of this milestone kindly rectify it before merging your the pull request.